planner/core/planbuilder.go

```diff
@@ -558,6 +558,19 @@ func (b *PlanBuilder) getPossibleAccessPaths(indexHints []*ast.IndexHint, tblInf
 		}
 
 		hasScanHint = true
+		if hint.IndexNames == nil {
```

Comment:
Could you also consider the comment style hint like `index(t)`?

Reply:
fixed